proposals/cri-o.adoc

```diff
@@ -0,0 +1,221 @@
+== CRI-O
+
+*Name of project*: cri-o
+
+*Description*:
+
+Cri-o is a stable container runtime specifically tailored for Kubernetes.
```

Comment:
> Cri-o

nit: CRI-O or cri-o ?

Reply:
ha, I think there isn't a standard for this. Perhaps CRI-O or cri-o ("Cri-o" never felt right)

Reply:
Suggest `cri-o`

Reply:
@rhatdan is there an official case size for Proper Noun cri-o?

Reply:
We define it as **C**ontainer **R**untime **I**nterface for **O**CI,  It should be capitalized.

Reply:
done